src/content/doc-surrealdb/deployment/fly.mdx

```diff
@@ -42,7 +42,7 @@ In the `Dockerfile` we specify which base image we want to use, and to which add
 ```bash title="Docker file"
 FROM surrealdb/surrealdb:latest
 EXPOSE 8080
-CMD ["start", "--bind", "0.0.0.0:8080", "file://data/srdb.db"]
+CMD ["start", "--bind", "0.0.0.0:8080", "surrealkv://data/srdb.db"]
```

Comment:
@leodar163 thanks for opening this issue. Since SurrealKV is in Beta we are refraining from recommending this for production use. Although there is a deprecation warning when using the `file://` prefix the deployment should work as expected. I can however add a note to that effect here so people are aware that you can use `surrealkv://` or `rocksdb://`